### ISSUE TITLE
Create a cli for handling tasks.

### ISSUE THREAD

**Issue by VandeurenGlenn** [VandeurenGlenn/Unclutter#3]
Now Unclutter is setup to run its tasks when the ui is running, splitting the app would have following benefits:
- Runs tasks in the background
- Is not part of the app process (so the app itself doesn't get blocked by the tasks while rendering, so the app will stay responsive...).
## Expected Cli behavior
- Creates a progress file that can be viewed in the ui
- Creates an index of paths found so tasks can be rerun without creating a new watcher that listens to 'add' events, this is way faster & takes less resources ...
- Creates a progress file containing the paths that were handled (user task runned once).
